example/docs/drop.cpp

```diff
@@ -0,0 +1,20 @@
+
+// Copyright (c) 2023 Jiri Nytra (jiri.nytra at gmail dot com)
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+#include <flux.hpp>
+
+#include <cassert>
+#include <vector>
+
+int main()
+{
+    std::vector vec{1, 2, 3, 4, 5};
+
+    auto dropped = flux::from(vec).drop(3);
```

Comment:
At the moment I'm only using the free-function versions of algorithms in examples -- I still need to decide the best way of documenting the member-function versions.

```suggestion
    std::vector vec{1, 2, 3, 4, 5};

    auto dropped = flux::drop(std::move(vec), 3);
```